contrib/libaacs/template.py

```diff
@@ -0,0 +1,30 @@
+pkgname = "libaacs"
+pkgver = "0.11.1"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--disable-optimizations"]
+hostmakedepends = [
+    "automake",
+    "bison",
+    "flex",
+    "libtool",
+    "pkgconf",
+]
+makedepends = ["libgcrypt-devel"]
+pkgdesc = "Research implementation of the AACS specification"
+maintainer = "miko <mikoxyzzz@gmail.com>"
+license = "LGPL-2.1-or-later"
+url = "https://www.videolan.org/developers/libaacs.html"
+source = f"https://download.videolan.org/pub/videolan/libaacs/{pkgver}/libaacs-{pkgver}.tar.bz2"
+sha256 = "a88aa0ebe4c98a77f7aeffd92ab3ef64ac548c6b822e8248a8b926725bea0a39"
+hardening = ["cfi", "sst", "vis"]
```

Comment:
this is a library and sst does not even work for DSOs

Reply:
well, i guess there's progs, but we don't use this anywhere yet for reasons i forgot

Reply:
i just nuked the entire hardening thing if that's fine

Reply:
if vis/cfi works then you should leave it?

easy way to test: see if the library linked into libbluray actually rips stuff

Reply:
mmmm i'll test tomorrow

Reply:
vis works, cfi doesn't